spec.bs

```diff
@@ -619,6 +625,12 @@ members:
         merchant). Optional, may be provided alongside or instead of
         {{SecurePaymentConfirmationRequest/payeeName}}.
 
+    :  <dfn>facilitatingEntityLogos</dfn> member
+    :: An optional list of logos representing entities that are facilitating
+        the payment that this SPC call is for, in descending order of
+        importance. The user agent is not required to show any or all of the
+        logos, see the [[#sctn-steps-to-check-if-a-payment-can-be-made]].
+
```

Comment:
I think it could be useful here to set some expectations for the caller that, although order matters, the specification is intentionally not requiring a particular layout of the logos. For example:

Note: It is intentional that this specification does not specify exactly how logos will be rendered so that the user agent can make a best effort in light of many considerations including the payment method, device considerations, and logo dimension considerations. The order of logos in the list does matter, however, which means that developers can experiment to determine whether a particular order gives desired results.

Reply:
Done, thanks! Slightly modified from your suggested text so please check it over.